include/SFML/Graphics/PrimitiveType.hpp

```diff
@@ -43,7 +43,7 @@ enum PrimitiveType
     LineStrip,     //!< List of connected lines, a point uses the previous point to form a line
     Triangles,     //!< List of individual triangles
     TriangleStrip, //!< List of connected triangles, a point uses the two previous points to form a triangle
-    TriangleFan    //!< List of connected triangles, a point uses the common center and the previous point to form a triangle
+    TriangleFan //!< List of connected triangles, a point uses the common center and the previous point to form a triangle
```

Comment:
That's kind of odd, but I can live without perfection 😄

Reply:
I looked into that. I believe the problem is that the comment on line 46 is so long that it exceeds our line limit so clang-format removes those alignment spaces to make sure the line is as short as possible. If we want better vertical alignment, we need shorter identifiers and/or shorter comments _or_ increase the maximum column width.